dev-perl/Gtk3-ImageView/Gtk3-ImageView-4.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DIST_AUTHOR=RATCLIFFE
+inherit perl-module virtualx
+
+DESCRIPTION="Image viewer widget for Gtk3"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-perl/Cairo
+	>=dev-perl/glib-perl-1.210
```

Comment:
The only version available is glib-perl-1.329.100.

Reply:
That's fine. Explicit dependency doesn't hurt, but has benefits of e.g. self-documenting and not breaking overlays